functional-tests/fn_prover_l1_dispatch.py

```diff
@@ -16,11 +22,13 @@ def main(self, ctx: flexitest.RunContext):
         prover_client_rpc = prover_client.create_rpc()
 
         # Wait for the Prover Manager setup
-        time.sleep(60)
+        time.sleep(5)
 
-        # Dispatch the prover task
-        task_id = prover_client_rpc.dev_strata_proveBtcBlock(1)
-        self.debug(f"got the task id: {task_id}")
+        # Prove L1 blocks from START_BLOCK to END_BLOCK
+        task_id = prover_client_rpc.dev_strata_proveBtcBlocks(
+            (L1_SCAN_PROVER_PARAMS["start_block"], L1_SCAN_PROVER_PARAMS["end_block"])
+        )
+        print("got the task id: {}", task_id)
```

Comment:
I think this can use the structured logging now.

Reply:
Context: #519.